### ISSUE TITLE
3.29.2 breaks TypeScript builds

### ISSUE THREAD

**Issue by vongohren** [twilio/twilio-node#429]
*Note: These issues are for bugs and feature requests for the helper libraries.
If you need help or support, please email help@twilio.com and one of our experts
will assist you!*

**Version:**
3.29.2

### Code Snippet
Just import it into a project using tcs as build command

### Exception/Log
```
api: $ tsc
api: ../../node_modules/twilio/lib/webhooks/webhooks.d.ts(1,35): error TS7016: Could not find a declaration file for module 'express'. '/code/api/node_modules/express/index.js' implicitly has an 'any' type.
api:   Try `npm install @types/express` if it exists or add a new declaration (.d.ts) file containing `declare module 'express';`
api: error Command failed with exit code 1.
```

### Steps to Reproduce
1. Create a new TSC lib
2. Import twilio 3.29.2
3. Build with tsc

**Comment by vongohren** [comment]
Should I have to install a child dependencies type? With my logic, I would say no.
There is something gone wrong in the build process.
And it seems this is the reason it happens: https://github.com/twilio/twilio-node/pull/406/files
It was introduced in 3.29.2

**Comment by kroleg** [comment]
@vongohren you're correct. It's my fault that it is broken now.
More info about where to put @types/* packages (dependencies or devDependencies) is here https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html

TL;DR `@types/express` should be in `dependencies`
